pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1267,7 +1267,7 @@
         "4.1.0.0": "4.1.0",
         "4.1.1.0": "4.3.0",
         "4.2.0.0": "4.4.0",
-        "4.4.0.0": "4.5.0"
+        "4.5.0.0": "4.5.0"
```

Comment:
This is now wrong. Did you update it manually? Try running an update from the pkg project build `msbuild /t:UpdatePackageIndex`

Reply:
No run it from inside the src\System.Data.SqlClient\pkg directory. You will need to do at least one full all configuration build from the root `build.cmd -allconfigurations` in order to enable individual package building.

Reply:
I did run with ```build.cmd -allConfigurations ``` command in the root directory and then ran the ```msbuild /t:UpdatePackageIndex``` command on SqlClient package directory, still seeing the failure.  The changes in the packageIndex.json file is also not related to versions.

Reply:
OK just manually update this section to be:

```
        "4.4.0.0": "4.5.0",
        "4.5.0.0": "4.6.0"
```